library/ui-styles/src/main/res/values/theme_dark.xml

```diff
@@ -46,11 +46,12 @@
         <!-- Presence Indicator colors -->
         <item name="vctr_presence_indicator_offline">@color/vctr_presence_indicator_offline_dark</item>
 
-        <!-- Some alias -->
+        <!-- Some aliases -->
         <item name="vctr_header_background">?vctr_system</item>
         <item name="vctr_list_separator">?vctr_content_quinary</item>
         <item name="vctr_list_separator_system">?vctr_system</item>
         <item name="vctr_list_separator_on_surface">?vctr_system</item>
+        <item name="vctr_unread_background">?vctr_content_tertiary</item>
```

Comment:
:+1:  for the alias, also allows light/dark to use separate theme values if needed